query/config.go

```diff
@@ -90,8 +91,11 @@ func (t *ConfigSummaryRequest) changesJoin() string {
 	return output + strings.Join(clauses, " AND ")
 }
 
-func (t *ConfigSummaryRequest) plainSelectClause(appendSelect ...string) []string {
-	return append(t.GroupBy, appendSelect...)
```

Comment:
I'm pretty sure the bug was happening here but not 100%.

What was happening is that the select columns generated by this function were internally being mutated by gorm.

Example: the first call returns columns 
`first := ["type", "namespace", "region", "jsonb_object_agg(health_grouped.health, count)::jsonb AS health"]`
and the second call returns 
`second := ["type", "namespace", "region", "json_object_agg(analysis_type, count)::jsonb AS analysis"]`

after the second call, `first` becomes idential to `second`